config/routes.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.routes.draw do
+  get 'home/index'
+  resources :payments
+  resources :categories
```

Comment:
- [ ] I would recommend allowing only necessary routes which you are using.

Example:

` resources :payments, except: [:edit, :show]`